bokehjs/test/unit/base.ts

```diff
@@ -105,6 +105,7 @@ describe("default model resolver", () => {
       "CustomJSFilter",
       "CustomJSHover",
       "CustomJSTickFormatter",
+      "CustomJSTicker",
```

Comment:
@mattpap FYI when the test for this fails it's really hard to debug without a proper object diff (there's just *"<50 line value> expected to equal <some other 50 line value>"*)

Reply:
This has been an annoyance to me as well on multiple occasions. I'm working on incorporating an object diff tool into our testing framework to resolve this.